src/views/transaction/components/transaction-form/transaction-form.view.jsx

```diff
@@ -383,7 +383,7 @@ function TransactionForm ({
       case TxType.Transfer: {
         const accountChecks = [
           getAccounts(receiver, [account.token.id]),
-          getCreateAccountAuthorization(receiver).catch(() => {})
+          !isHermezBjjAddress(receiver) ? getCreateAccountAuthorization(receiver).catch(() => {}) : {}
```

Comment:
I think that instead of adding an empty object, not adding anything at all would be more "correct". I propose something like this:

```js
...(  !isHermezBjjAddress(receiver) ? [getCreateAccountAuthorization(receiver).catch(() => {})] : []  )
```

What do you think?

Reply:
Done